### ISSUE TITLE
PART A - Classification by fine-tuning BERT 

### ISSUE THREAD

**Issue by efemeryds** [efemeryds/offensive-language-detection#4]
Run your notebook on colab, which has (limited) free access to GPUs.
You need to enable GPUs for the notebook:

● navigate to Edit → Notebook Settings
● select GPU from the Hardware Accelerator drop-down

➢ Install the simpletransformers library: !pip install simpletransformers 
(you will have to restart your runtime after the installation) 

➢ Follow the documentation to load a pre-trained BERT model: ClassificationModel('bert', 
'bert-base-cased') 

➢ Fine-tune the model on the OLIDv1 training set and make predictions on the OLIDv1 test 
set (you can use the default hyperparameters). Do not forget to save your model, so that 
you do not need to fine-tune the model each time you make predictions. 
If you cannot fine-tune your own model, contact us to receive a checkpoint.
a. Provide the results in terms of precision, recall and F1-score on the test set and provide 
a confusion matrix (2 points)

Compare your results to the baselines and to the results described in the paper in 2–4 
sentences
